addons/project/models/project_task.py

```diff
@@ -351,7 +351,7 @@ def _compute_display_in_project(self):
             )
 
     def _inverse_parent_id(self):
-        for task in self:
+        for task in self.sudo():
             if not task.parent_id:
                 task.display_in_project = True
             elif task.display_in_project and task.project_id == task.parent_id.project_id:
```

Comment:
I guess the sudo is only useful to get the project set on the parent task, right? If yes then we could add the sudo just there, no? 🙂

Reply:
It is also needed to set  the `display_in_project` field, this is why it is there.